### PR TITLE
[Feature](export) Support cancel export statement

### PR DESCRIPTION
**Pull request by dutyu**
# Proposed changes

Issue Number: close #xxx

## Problem summary

Users can cancel a running/pending export job by using 
`
CANCEL EXPORT [FROM db] WHERE [LABEL = "export_label" | LABEL like "label_pattern" | STATE = "PENDING/EXPORTING"]
`

## Checklist(Required)

1. Does it affect the original behavior: 
    - [ ] Yes
    - [x] No
    - [ ] I don't know
2. Has unit tests been added:
    - [x] Yes
    - [ ] No
    - [ ] No Need
3. Has document been added or modified:
    - [ ] Yes
    - [x] No
    - [ ] No Need
4. Does it need to update dependencies:
    - [ ] Yes
    - [x] No
5. Are there any changes that cannot be rolled back:
    - [ ] Yes (If Yes, please explain WHY)
    - [x] No

## Further comments

**I will add some relevant docs later.**

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by hello-stephen** [comment]
TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 35.43 seconds
 load time: 431 seconds
 storage size: 17123356879 Bytes
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20221216063313_clickbench_pr_64329.html

**Review by hf200012 — COMMENTED** [review]
LGTM

**Review by hf200012 — APPROVED** [review]
LGTM

File: fe/fe-core/src/main/java/org/apache/doris/load/ExportMgr.java [hunk 1]
```diff
@@ -294,12 +351,12 @@ public void replayCreateExportJob(ExportJob job) {
     }
 
     public void replayUpdateJobState(long jobId, ExportJob.JobState newState) {
-        writeLock();
+        readLock();
```
[the remark this answers is not in the document]
Comment (dutyu) [reply fe/fe-core/src/main/java/org/apache/doris/load/ExportMgr.java:354]: This lock is designed to protect `idToJob` and `labelToJobId`, `replayUpdateJobState` method just read from the `idToJob`, so use a read lock may be more reasonable.

**Pull request merged**